src/main/java/de/zalando/zmon/dataservice/Application.java

```diff
@@ -1,18 +1,28 @@
 package de.zalando.zmon.dataservice;
 
+import de.zalando.zmon.dataservice.config.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
+import io.opentracing.NoopTracerFactory;
+import io.opentracing.Tracer;
```

Comment:
Please remove unused imports (IntelliJ IDEA can do that for you :) you just need to enable "Optimise imports" in formatting options)